src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -529,7 +535,20 @@ export class PoolFormComponent implements OnInit {
             formControlName: 'erasureProfile',
             attr: 'name'
           },
-      { externalFieldName: 'rule_name', formControlName: 'crushRule', attr: 'rule_name' }
+      { externalFieldName: 'rule_name', formControlName: 'crushRule', attr: 'rule_name' },
+      {
+        externalFieldName: 'quota_max_bytes',
+        formControlName: 'max_bytes',
+        replaceFn: this.formatter.toBytes,
+        editable: true,
+        resetValue: this.editing ? 0 : undefined
```

Comment:
When editing a pool, clearing quota field should disable pool quota.

Reply:
I've rebased and found previous changes missed this recommendation: https://github.com/ceph/ceph/pull/27945#discussion_r281976639

When editing a pool, clearing quota fields should disable corresponding quotas.
So I think it's better to start another round of review and test again, since pool testing code on master changed too.